web/tests/functional/diff_local_remote/test_diff_local_remote.py

```diff
@@ -574,6 +574,38 @@ def test_diff_codeclimate_output(self):
 
         shutil.rmtree(export_dir)
 
+    def test_diff_no_trim_codeclimate_output(self):
+        """ Test codeclimate output when using diff and don't set env vars. """
+        base_run_name = self._run_names[0]
+
+        export_dir = os.path.join(self._local_reports, "export_dir")
+
+        diff_cmd = [self._codechecker_cmd, "cmd", "diff",
+                    "--unresolved",
+                    "--url", self._url,
+                    "-b", base_run_name,
+                    "-n", self._local_reports,
+                    "-o", "codeclimate",
+                    "-e", export_dir]
+
+        self.run_cmd(diff_cmd, self._env)
+        codeclimate_issues_file = os.path.join(export_dir,
+                                               'codeclimate_issues.json')
+        self.assertTrue(os.path.exists(codeclimate_issues_file))
+
+        with open(codeclimate_issues_file, 'r',
+                  encoding="utf-8", errors="ignore") as rw_file:
+            issues = json.load(rw_file)
+
+        malloc_issues = [i for i in issues if i["check_name"] == "unix.Malloc"]
+        self.assertNotEqual(len(malloc_issues), 0)
+
+        file_path = malloc_issues[0]["location"]["path"]
+        self.assertTrue(os.path.isabs(file_path))
+        self.assertTrue(file_path.endswith(f"/new_delete.cpp"))
+
+        shutil.rmtree(export_dir)
```

Comment:
Cleaning required on all exit paths, for example when the issues file could not open and throws an exception.

Reply:
The whole workspace will be cleaned up at the end of this functional test:
https://github.com/Ericsson/codechecker/blob/6788bf4e730551c12e83d4ac553b901f556bda07/web/tests/functional/diff_local_remote/__init__.py#L149